Explorer/Assets/DCL/Interaction/Raycast/Systems/InitializeRaycastSystem.cs

```diff
@@ -28,7 +37,18 @@ protected override void Update(float t)
         private void HandleNewComponent(in Entity entity)
         {
             var comp = new RaycastComponent();
-            World.Add(entity, comp);
+            PBRaycastResult? raycastResult = raycastComponentPool.Get();
+            World.Add(entity, comp, raycastResult);
+        }
+
+        [Query]
+        [All(typeof(TransformComponent), typeof(PBRaycast))]
+        [None(typeof(PBRaycastResult))]
+        private void HandleMissingRaycastResult(in Entity entity)
+        {
+            //I Dont like this, but the SDK removes the PBRaycastResult and does not add it when it adds the PBRaycast + we dont remove the RaycastComponent
```

Comment:
the SDK never adds the `PBRaycastResult` that's a responsibility of the Explorer/Renderer/Engine since the physics run in there...